components/builder-api/src/depot/server.rs

```diff
@@ -428,6 +428,19 @@ pub fn origin_member_delete(req: &mut Request) -> IronResult<Response> {
     }
 }
 
+fn check_forced(req: &mut Request) -> bool {
+    match helpers::extract_query_value("forced", req) {
```

Comment:
could probably simplify this function to:
```
if let Some(flag) = helpers::extract_query_value("forced", req) {
  flag == "true"
} else {
  false
}
```

Reply:
Ah clever!